### DISCUSSION TITLE
is it possible to use the same license key in another virtual environment on the same PC

### DISCUSSION THREAD

**Discussion by kittycat-ribbon** [dashingsoft/pyarmor#1905] [Q&A]
After installing a Pyarmor license key in one virtual environment, 
is it possible to use the same license key in another virtual environment on the same PC?

**Comment by jondy** [comment]
By default all the pyarmor in the same machine will share the license key which in the path `~/.pyarmor`
Please check the usage of option `--home` for more
https://pyarmor.readthedocs.io/en/latest/reference/man.html#cmdoption-pyarmor-home
